configs/components/openssl.rb

```diff
@@ -65,12 +84,9 @@
     pkg.environment "CC" => "/opt/pl-build-tools/bin/gcc"
   elsif platform.is_windows?
     target = platform.architecture == "x64" ? "mingw64" : "mingw"
-    pkg.environment "PATH" => "#{settings[:gcc_bindir]}:$$PATH"
+    pkg.environment "PATH" => "$$(cygpath -u #{settings[:gcc_bindir]}):$$PATH"
     pkg.environment "CYGWIN" => settings[:cygwin]
     pkg.environment "CC" => settings[:cc]
```

Comment:
Yeah, I think we do. I tried it without CC, but got failures when some stuff later in the build couldn't find gcc :(